packages/suite/src/reducers/suite/notificationReducer.ts

```diff
@@ -25,19 +27,10 @@ export interface NotificationEntry {
 
 export type State = NotificationEntry[];
 
-const initialState: State = [
-    // {
-    //     id: undefined,
-    //     type: "info",
-    //     title: "Some static notification",
-    //     message: "This one is not cancelable",
-    //     cancelable: false,
-    //     actions: []
-    // }
-];
+const initialState: State = [];
 
-const addNotification = (state: State, payload: any) => {
-    return state.push({
+const addNotification = (draft: Draft<State>, payload: NotificationAddPayload) => {
+    draft.push({
         key: new Date().getTime().toString(),
```

Comment:
what a lovely random number generator 😀

Reply:
> what a lovely random number generator 😀
:D https://www.npmjs.com/package/uuid ?

Reply:
do you think we are ever going to have id collision?

Reply:
> do you think we are ever going to have id collision?

With the same time? yes

Reply:
I don't think it could really happen, we would have to dispatch multiple notifications in the same millisecond, there is certainly some time cost for dispatch itself that's going to be more than 1ms. But I still don't feel good about stuff like this. (or user could go back in time (os setting) and try to make a collision on purpose!) 😁

Reply:
anyway we can resolve it in another PR if we find it necessary.

Reply:
Yes, and I actually did not touch the "key" logic at all and took it from wallet, so it is probably "battle tested" already.

Reply:
Id say, there cant be collisions. Even if we dispatch 2 actions in single ms, resolving 1 dispatch takes more then 1ms. And we are in a single threaded lanaguage so I am ok with it.